reference-implementation/.eslintrc.json

```diff
@@ -160,7 +160,16 @@
     "id-blacklist": "off",
     "id-length": "off",
     "id-match": "off",
-    "indent": ["error", 2, { "SwitchCase": 1 }],
+    "indent": ["error", 2, {
+      "SwitchCase": 1,
+      "MemberExpression": 2,
+      "FunctionDeclaration": { "parameters": "first" },
```

Comment:
The new rules seem reasonable... @domenic, what do you think?

I don't understand why the indentation of `.catch(rethrowAssertionErrorRejection);` expressions has changed, although they always seemed like a weird special case to me anyway.

Reply:
The `.catch` indentation is controlled by `MemberExpression`. I could set it to `0` to keep the current style, but then it might look weird in other places...

Alternatively, I can change all of those:
```js
promise.then(() => {
  /* ... */
})
.catch(rethrowAssertionErrorRejection);
```
to:
```js
promise.then(() => {
  /* ... */
}).catch(rethrowAssertionErrorRejection);
```
and avoid the problem entirely. 😛

Reply:
> Alternatively, I can change all of those:
> 
> ```js
> promise.then(() => {
>   /* ... */
> })
> .catch(rethrowAssertionErrorRejection);
> ```
> 
> to:
> 
> ```js
> promise.then(() => {
>   /* ... */
> }).catch(rethrowAssertionErrorRejection);
> ```

I like this idea, but let's see what @domenic says. These statements predate my involvement in the project.

Reply:
I'm happy to change in that way!

Reply:
All right, I'll push one more commit to reformat the chained method calls. 🙂